### PR TITLE
Project Status Page

### PR DESCRIPTION
**Pull request by dwelch2344**
With Protractor becoming LTS and given the partnership with [HeroDevs](https://www.herodevs.com) to provide support past August 2023, we had discussed adding documentation to explain what the future holds. 

Happy to collaborate on the messaging, style, or any other aspect! We just wanted to take a swing at it to help with the lift! 

Screen shot of change:
Screen Shot 2023-01-27 at 4 27 39 PM

**Note:** This PR builds on #5556 (as it was required to run the site for review)

### CODE REVIEW

**Comment by dwelch2344** [comment]
Just pushed some content fixes. Latest banner & status page image below

Screen Shot 2023-04-10 at 1 56 41 PM
Screen Shot 2023-04-10 at 1 57 09 PM

File: website/gulpfile.js [hunk 1]
```diff
@@ -46,9 +46,10 @@ gulp.task('dgeni', function(done) {
     var isBrowser = function(item) {
       return item.alias == 'browser';
     };
-    if (!toc.items.some(isBrowser)) {
-      return Promise.reject('Generated toc.json missing docs for Protractor function.');
-    }
+    // TODO: dtw - find out what this browser check is really for. May be obsolete
```
Comment (mgechev) [inline website/gulpfile.js:49]: We'd not want to disable build time checks

File: website/partials/status.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Although active development on Protractor ended in December of 2022, the project will continue to be supported under <strong>Long Term Support</strong> and will continue to receive critical security &amp; compatibility fixes via the community through August 2023. This is important as there are many actively developed projects still benefiting from their existing code coverage, and we anticipate Protractor to continue functioning well into the future. 
+  </p>
+  <p>
+    In short: Protractor's current status is <strong>Long Term Support</strong> and <strong>will End Of Life in September 2023</strong>.
+  </p>
+  
+  <h2>What does End Of Life mean?</h2>
+  The packages will remain accessible on GitHub, npm, Bower, and Release archive. This website will also remain here indefinitely. The project will be archived on GitHub and remain accessible indefinitely, but will no longer be open to issues / comments / etc. 
+  
+  <h2>Support After August 2023</h2>
```
Comment (mgechev) [inline website/partials/status.html:15]: Can we change the page to look like? https://docs.angularjs.org/misc/version-support-status

## Extended Long Term Support

If you need extended support for Protractor, you should consider:
- [herodevs.com](https://herodevs.com)

**Review by mgechev — CHANGES_REQUESTED** [review]
Left suggestions inline

File: website/partials/status.html (hunk 2 above)
Comment (aaronfrost) [reply website/partials/status.html:15]: @mgechev given that HeroDevs is supporting protractor for an additional year at no cost, it seems that it makes sense to explain the free year of NES here. I feel like that would make sense. Should we have a call to talk about this?

**Comment by dwelch2344** [comment]
Given the website is currently deployed off the 5.4 branch, I migrated the work to #5560. All the comments should be addressed, but feel free to raise any issue there.

**Pull request closed without merging**